lib/awful/client.lua.in

```diff
@@ -71,7 +71,7 @@ function client.jumpto(c, merge)
     end
 
     -- focus the client
-    capi.client.focus = c
+    c:emit_signal("request::activate", "client.jumpto")
     c:raise()
```

Comment:
The `c::raise` should be removed.  That's what the signal handler is supposed to to - see `ewmh.activate`.